statusboard/src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,23 @@
+.App-footer {
+  width: 100%;
+  min-height: 5rem;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-size: 1.25rem;
+  background-color: var(--cfa-purple-dark);
+
+  p {
+    color: var(--cfa-white);
+    font-family: var(--font-family-text);
+    margin-left: 1.0rem;
+    margin-right: 1.0rem;
```

Comment:
The links in here have a hover color that blends into the background. Changing that could be a more accessibility friendly improvement.